app/views/boards/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<body>
+	<div class="board">
+ 		<table class="one">
+ 			<% 4.times do %>
+ 			<tr>
+ 		 		<td class="black"></td>
```

Comment:
Hmmm now that I think about it - maybe code complication depending on the way you look at it lol